db/seeds.rb

```diff
@@ -369,3 +369,12 @@
     name: english_language_provider[:name],
   ).update!(english_language_provider.except(:name))
 end
+
+subject_limited_countries = %w[GH IN JM NG SG ZA ZW]
+
+subject_limited_countries.each do |country_code|
+  country = Country.find_by(code: country_code)
+  next unless country
+
+  country.update!(subject_limited: true)
+end
```

Comment:
```suggestion
Country.where(code: subject_limited_countries).update_all(subject_limited: true)
```

This avoids a database query for each country, and instead makes a single query that updates all the countries in one go.